src/cogent3/core/alignment.py

```diff
@@ -2013,6 +2013,22 @@ def __init__(self, map, data, length=None):
         if hasattr(data, "name"):
             self.name = data.name
 
+    def annotate_matches_to(
+        self, pattern=None, annot_type=None, name=None, allow_multiple=False
+    ):
+        """Creates an annotation at the specified pattern in a sequence."""
+        from cogent3.core.annotation import Feature
```

Comment:
All comments about method for this class should be applied to the Sequence class. This method should directly call that one, i.e. `return self.data.annotate_matches_to(...)`

remove this, just use `add_feature` method below